dir.props

```diff
@@ -0,0 +1,307 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
```

Comment:
In the CLI, we've been following a pattern of splitting these properties up into smaller .props files.  That way "dir.props" doesn't become a dumping ground of a bunch of unrelated properties.

How do you feel about following that pattern here?